test/tests/neutronics/moving_mesh/mesh_tally.i

```diff
@@ -0,0 +1,133 @@
+[GlobalParams]
```

Comment:
It seems like this input file has a lot of similarities with the call_tally.i input file. Do you think it would make sense to generate a base input file and utilize the `!include` capabilities to reduce duplication?